data/lua/cave_map_generator.lua

```diff
@@ -133,8 +132,9 @@ function callbacks.generate_map(params)
 				end
 				return res
 			end
-			local path, cost = wesnoth.find_path(v.start_x, v.start_y, v.dest_x, v.dest_y, calc, params.map_width, params.map_height)
-			for i, loc in ipairs(path) do
+			local path = wesnoth.find_path(
```

Comment:
I don't really like splitting up long lines like this...

Reply:
FTR, I prefer splitting them.

Reply:
@CelticMinstrel the line is just too damn long. You can't have two such lines in a side-by-side comparison, for example. There are strict guidelines on many serious projects that just forbid long lines, and that's it. These conventions are invented not out of thin air, but are instead summarized from a large number of reasons, polished by generations of developers.

The only question there is in practice -- which constant to use, exactly? Some prefer 80, some prefer 100, some prefer 120. I never saw any guidelines suggesting use of more than 120.

Reply:
I can google out and put lots of references, if needed, of course. Like Rust language guidelines, python styling, code styles in Twitter, Google, etc etc.